docker/Dockerfile.ci

```
@@ -58,8 +58,9 @@ RUN groupadd -g 1500 rust \
   && echo 'rust ALL=NOPASSWD: ALL' >> /etc/sudoers.d/10-rust \
   && echo 'Defaults    env_keep += "DEBIAN_FRONTEND"' >> /etc/sudoers.d/env_keep
 
-# Ensure we are ready with cargo hakari
+# Ensure we are ready with cargo hakari and cargo deny
 RUN cargo install cargo-hakari
+rUN cargo install cargo-deny
```

... guess this isn't case sensitive? It looks weird though...

```suggestion
RUN cargo install cargo-deny
```

---

🤦